src/core/crypto.js

```diff
@@ -32,15 +32,14 @@ class ARCFourCipher {
     this.a = 0;
     this.b = 0;
     const s = new Uint8Array(256);
-    let i,
-      j = 0,
-      tmp,
-      keyLength = key.length;
-    for (i = 0; i < 256; ++i) {
+    const keyLength = key.length;
+    let j = 0;
```

Comment:
Nit: You could *maybe* just move this definition to the second loop below (at line 41), since it's only used within that loop unless I'm missing something here:
```javascript
for (let i = 0, j = 0; i < 256; ++i) {
```